contributing-device-support.md

```diff
@@ -1,19 +1,25 @@
-# Custom Board Support
+# Customer Board Support
+
+There are several options for onboarding the hardware onto balenaOS:
+
+1. You can find another board that would work for your case that is supported in our [supported device list](https://www.balena.io/docs/reference/hardware/devices/).
+2. You can consider our [Custom Device Support (CDS)](https://www.balena.io/pricing/#custom-device-support), which is a paid service during which we onboard the device for you and maintain it for a monthly fee. [Here](https://drive.google.com/file/d/17-DTeXkflkzW8Qduj48ut7JNUTWqiRv-/view?usp=sharing) is an overview including pricing.
+3. If you have folks on your team who are familiar with Yocto, then you could build and maintain it yourselves using our customer board support (CBS) documentation. Follow along below!
 
 **Pre-requisites:** a [Yocto](https://www.yoctoproject.org) Board Support Package (BSP) layer for your particular board. It should be compatible with the Yocto releases balenaOS supports.
```

Comment:
```suggestion
## Pre-requisites

A [Yocto](https://www.yoctoproject.org) Board Support Package (BSP) layer for your particular board. It should be compatible with the Yocto releases balenaOS supports.
```

Since the para above becomes intro, I feel like we need a pre-requistes para